datasets/swedish_reviews/swedish_reviews.py

```diff
@@ -0,0 +1,81 @@
+# coding=utf-8
+# Copyright 2020 The TensorFlow Datasets Authors and the HuggingFace Datasets Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Lint as: python3
+
+from __future__ import absolute_import, division, print_function
+
+import csv
+import os
+
+import datasets
+
+
+_DOWNLOAD_URL = "https://raw.githubusercontent.com/timpal0l/swedish-sentiment/main/swedish_sentiment.zip"
+_TRAIN_FILE = "train.csv"
+_VAL_FILE = "dev.csv"
+_TEST_FILE = "test.csv"
+
+_CITATION = ""
```

Comment:
no citation ?

Reply:
@lhoestq Not sure what to write there since I have scraped the dataset myself, there is not paper related to it. So not sure what to write there and on Licensing, that's why I left em blank.